python/send_message/README.md

```diff
@@ -0,0 +1,102 @@
+# SendMessage()
+
+A Python Cloud Function for sending a message using a specific channel to a receiver
+
+Supported channels are `SMS`, `EMAIL` ,`DISCORD` and `TWITTER`.
+
+_SMS Example payload_
+
+```json
+{ "type": "SMS", "receiver": "+123456789", "message": "Programming is fun!" }
+```
+
+_Email Example payload_
+
+```json
+{
+  "type": "EMAIL",
+  "receiver": "hello@example.com",
+  "message": "Programming is fun!",
+  "subject": "Programming is funny!"
+}
+```
+
+_Discord Example payload_
+
+```json
+{ "type": "DISCORD", "message": "Hi" }
+```
+
+_Twitter Example payload_
+
+```json
+{ "type": "TWITTER", "receiver": "", "message": "Programming is fun!" }
+```
+
+_Successful function response:_
+
+```json
+{ "success": true }
+```
+
+_Error function response:_
+
+```json
+{
+  "success": false,
+  "error": "Failed to send message,check webhook URL"
+}
+```
+
+## 📝 Variables
+
+List of environment variables used by this cloud function:
+
+Mailgun
+
+- **MAILGUN_API_KEY** - API Key for Mailgun
+- **MAILGUN_DOMAIN** - Domain Name from Mailgun
+
+Discord
+
+- **DISCORD_WEBHOOK_URL** - Webhook URL for Discord
+
+Twilio
+
+- **TWILIO_ACCOUNT_SID** - Acount SID from Twilio
+- **TWILIO_AUTH_TOKEN** - Auth Token from Twilio
+- **TWILIO_SENDER** - Sender Phone Number from Twilio
+  Twitter
```

Comment:
Same issue here as in https://github.com/open-runtimes/examples/pull/44, needs space around this to make it display properly

Reply:
yeah fixed